tracer/build/_build/Build.cs

```diff
@@ -143,6 +143,8 @@ public Build()
                 DeleteReparsePoints(SourceDirectory);
                 DeleteReparsePoints(TestsDirectory);
             }
+
+            RootDirectory.GlobDirectories("obj*").ForEach(x => DeleteDirectory(x));
```

Comment:
the `obj...` folder is well known right? I kind of prefer to use the exact name than deleting `obj*` in the root folder.

Reply:
oh my, didn't know about the plain `obj` folder. I thought we could use something like `obj_arm64*`

Reply:
And where I was about to add an `objects` folder :P

Reply:
This just exemplifies that we should be using the artifacts layout, not dumping crap in the root folder. i.e. "make native do what the managed side does"